RELEASE/scripts/autoscend/combat/auto_combat_default_stage5.ash

```diff
@@ -173,6 +173,13 @@ string auto_combatDefaultStage5(int round, monster enemy, string text)
 		return useSkill($skill[Stuffed Mortar Shell]);
 	}
 
+	
+	//Roman Candelabra red candle
+	if(have_equipped($item[Roman Candelabra]) && have_effect($effect[Everything Looks Red]) == 0)
```

Comment:
```suggestion
	if(have_equipped($item[Roman Candelabra]) && have_effect($effect[Everything Looks Red]) == 0 && !auto_haveDarts())
```

It's unlikely to happen since the Bullseye handler is in auto_combat_default_stage2.ash but just in case we probably want people with the Dart Holder not to use this if it does slip through to here.